### ISSUE TITLE
Multiple labels are inconsistent

### ISSUE THREAD

**Issue by NewSun55** [PerrySkywalker/DAMLN#1]
<html>
<body>

0 | 201905519 | (1, 1, 1) | 202011445 | (1, 1, 0) | 54866.18 | (1, 1, 0)

3 | 1 | 43101.16 | (1, 1, 0) | 46308.17 | (1, 1, 0) | 202016666 | **(1, 1, 2)**

</body>
</html>

 each WSI Wi has multiple labels {yi,1, yi,2, ..., yi,M}, where yi,j ∈ {0, 1}

A stunning idea, thank you to the author for the open source. The value of multiple labels in the paper is 0, 1. However, flod.csv shows 1, 1, 2

**Comment by PerrySkywalker** [comment]
The data for this study were collected from hospital clinical cases, including a patient cohort with HER2 immunohistochemical (IHC) test results of 2+. According to the guidelines of the College of American Pathologists (CAP), HER2 2+ is considered a borderline value, and fluorescence in situ hybridization (FISH) testing is required for definitive determination. However, due to practical considerations in clinical practice, not all patients underwent confirmatory FISH testing, resulting in uncertainty regarding the HER2 status of these cases. In our data annotation, we employed a triplet (1,1,2) to represent estrogen receptor (ER) positive, progesterone receptor (PR) positive, and HER2 status undetermined. In the calculation of the loss function, we specifically excluded the loss associated with samples of undetermined HER2 status to ensure the accuracy and reliability of model training.

**Comment by NewSun55** [comment]
Thank you for your reply, it has resolved my question. Wishing everything all the best.
